src/test/rustdoc-js/summaries.js

```diff
@@ -0,0 +1,7 @@
+const QUERY = 'summaries';
```

Comment:
You can make multiple queries at once by using an array (then `EXPECTED` has to be an array as well).

Take a look at `doc-alias.js` if you're not sure how it should look like.